### PR TITLE
Updates to v2 encoding

### PR DESCRIPTION
**Pull request by codefromthecrypt**

### CODE REVIEW

File: backend.py [hunk 1]
```diff
@@ -15,6 +16,7 @@ def print_date(request):
 def main():
     settings = {}
     settings['service_name'] = 'backend'
+    settings['zipkin.encoding'] = Encoding.V2_JSON
```
Comment (codefromthecrypt) [inline backend.py:19]: wonder if transport.py could become a function of settings..

ex configure_transport(settings) which then sets the encoding and the handler?

**Comment by codefromthecrypt** [comment]
cc @drolando

File: backend.py (hunk 1 above)
Comment (codefromthecrypt) [inline backend.py:19]: wonder if transport.py could become a function of settings..

ex configure_transport(settings) which then sets the encoding and the handler?

**Comment by codefromthecrypt** [comment]
tested with the new UI also (just for fun)

`java -jar zipkin.jar  --zipkin.ui.source-root=classpath:zipkin-lens`

screen shot 2019-02-26 at 9 40 05 am

File: backend.py (hunk 1 above)
Comment (drolando) [reply backend.py:19]: yeah that's something I thought about as well. Even if you use the new `py_zipkin.transport.BaseTransportHandler`, the transport still doesn't know what's the encoding.

We could pass it as argument to `send`, but that'd be another backward incompatible api change.

**Review by drolando — APPROVED** [review]

**Comment by codefromthecrypt** [comment]
thanks for the tips. you have karma now also, in case anything I've done in this repo makes you cringe

**Pull request merged**